File: Bubble_Sort/Python/Bubble.py
def bubble(array):
	arr = len(array)
	
	for i in range(arr):
		for j in range(0, arr - i - 1):
			if array[j] > array[j+1]:
				array[j], array[j+1] = array[j+1], array[j]

def print(array):
	arr = len(array)
	
	print("[")
	for i in range(arr):
		if i == arr - 1:
			print(array[i] + "]")
		else:
			print(array[i] + ", ")

File: Bubble_Sort/JAVA/Bubble.java
import java.util.*;

public class Bubble extends Sorting {
	public void sort(int array []) {
		int arr = array.length;
		for (int i = 0; i < arr - 1; i++) {
			for (int j = 0; j < arr - i - 1; j++) {
				if (array[j] > array[j + 1]) {
					int tmp = array[j];
					array[j] = array[j+1];
					array[j+1] = tmp;
				}
			}
		}
	}
}